rules/windows/command_and_control_certutil_network_connection.toml

```diff
@@ -2,9 +2,35 @@
 creation_date = "2020/03/19"
 integration = ["endpoint", "windows"]
 maturity = "production"
-updated_date = "2023/01/31"
 min_stack_comments = "New fields added: required_fields, related_integrations, setup"
 min_stack_version = "8.3.0"
+updated_date = "2023/02/27"
+
+[transform]
+[[transform.osquery]]
```

Comment:
Just to confirm, we are unable to do the following?

```sql
[transform]
[[transform.osquery]]
retrieve_dns_cache = "SELECT * FROM dns_cache"
retrieve_all_services = ""SELECT description, display_name, name, path, pid, service_type, start_type, status, user_account FROM services"
```

This way we collapse this a bit and in the code can rely on `self.transform.osquery.retrieve_dns_cache`? Thoughts?

Reply:
we _could_ but I think it sacrifices understanding of the underlying data structure, since `label` and `query` are part of the query. But more importantly, it makes validation is bit less accurate and handling unknown keys a bit more complicated

Reply:
Fair point, if it adds complexity to validation no need to re-work as it does not add any resiliency to the code. Personally it just looks for aesthetically pleasing IMO 😅